pegasus/sites.v3/code.org/public/css/music-lab-skinny-banner.scss

```diff
@@ -0,0 +1,73 @@
+$width-sm: 640px;
+
+.music-lab-skinny-banner {
+  background: radial-gradient(
+      80% 200% at -10% 50%,
+      #02107d 5.27%,
+      rgba(33, 34, 64, 0) 100%
+    ),
+    radial-gradient(
+      50% 200% at 100% 50%,
+      rgba(87, 30, 161, 0.7) 0%,
+      rgba(32, 18, 73, 0) 100%
+    ),
+    #041214;
+  padding: 2rem;
+
+  .wrapper {
+    max-width: 960px;
+    margin: 0 auto;
+    gap: 4rem;
+
+    p.desc {
+      margin-bottom: 10px;
+    }
+
+    @media (max-width: 900px) {
+      gap: 2rem;
+    }
+
+    @media (max-width: $width-sm) {
+      padding-block: 2rem;
+      flex-direction: column;
+      justify-content: center;
+      gap: 1rem;
+    }
+  }
+
+  .text-wrapper {
+    flex: 2;
+
+    @media (max-width: $width-sm) {
+      text-align: center;
+    }
+  }
+
+  .partner {
+    p {
+      white-space: nowrap;
+    }
+
+    img {
+      max-width: 12rem;
+      margin-left: 0.5rem;
+    }
+
+    @media (max-width: $width-sm) {
+      justify-content: center;
+      flex-direction: column;
+
+      img {
+        margin: 0;
+      }
+    }
+  }
+
+  img {
+    max-width: 21.25rem;
+  }
```

Comment:
This size is too big on mobile (see screenshot of current UI), you can change it to this so it works better at all sizes:
```suggestion
  img {
    max-width: 18rem;
  }
```
<img width="431" alt="Screenshot 2024-10-28 at 11 35 31 AM" src="https://github.com/user-attachments/assets/5a3ab0ae-857c-4a94-ae5d-81140ad5373a">